docs/tutorial.rst

```diff
@@ -852,19 +895,28 @@ Thus, we have removed 6 singleton sites from the tree sequence.
 Working with Tables
 *******************
 
-.. todo::
-    This section is a work in progress and needs to be revised.
```

Comment:
I think this is still useful? Although you could make the argument it should be merged with one of the other sections on tables.